examples/quoted-identifiers/quoted_identifiers.bal

```diff
@@ -5,18 +5,54 @@ function 'function(int val) returns int {
     return val + 1;
 }
 
+// Use different kinds of characters in the name of the function.
+function 'get_ɱȇşşağę_for_\#(int val) returns string {
+    return val.toString() + " is a numeric value" ;
+}
+
+// Use different kinds of characters in the type definition.
+// The quoted identifier syntax is used in both the type name and the field names.
+type 'Person_\{name\&Ȧɢέ\} record {|
+    string 'first\ name;
+    int 'Ȧɢέ;
+|};
+
 public function main() {
 
     // Use the reserved keyword `int` as the name of the variable.
     int 'int = 1;
 
     // Invoke the function named `function` with the variable named `int`.
-    // The quoted identifier syntax is used to refer to both the function and
+    // The quoted identifier syntax is used in both the function and
```

Comment:
On second thoughts, I think what was originally there is correct. Sorry!

Shall we change this back to
```suggestion
    // The quoted identifier syntax is used to refer to both the function and
```